### ISSUE TITLE
Navigate namespaced keywords

### ISSUE THREAD

**Issue by devth** [gga/json-path#11]
Does json-path support navigating a Clojure data structure with namespaced keywords? Example:

```clojure
(jp/at-path "$.foo/qux" {:foo/qux {:bar "Hello, world!"}})
```

This returns `nil`. Tried this:

```
(jp/at-path "$['foo/qux'].bar" {:foo/qux {:bar "Hello, world!"}})
```

returns: 

```
CompilerException java.lang.Exception: object must be an array
```

**Comment by cburgmer** [comment]
Hey Trevor, if you look at what `parse-path` does to `"$.foo/qux"`, you'll notice that it doesn't return one child token, but two: 

```clj
[:path [[:root] [:child] [:key "foo"] [:key "qux"]]]
```

You can reproduce this e.g. by adding a line to parser_test.clj.
Now, this probably needs some investigation on whether this is required by the JsonPath spec, or can be adapted to work nicely with this use case.

**Comment by devth** [comment]
Ok, I see. Is it supposed to do that (intentional) or just how it happens to work currently?

Since namespaced keywords are common in Clojure I'd argue that it should support them in a natural way, like my first example.

**Comment by devth** [comment]
Added support via above PR. This is consistent with at least one other JsonPath implementation, http://jsonpath.com/.

**Comment by devth** [comment]
@gga or @cburgmer – would either of you be able to review the PR? #12

**Comment by cburgmer** [comment]
Thanks for the PR. I've merged it, added you as a contributor to README.md and released a new version under 1.0.1. Hope this works for you :)

**Comment by devth** [comment]
@cburgmer awesome, thanks!
